src/Request.js

```diff
@@ -159,6 +166,10 @@ define([
 			};
 		},
 
+		_getRequestOptions: function(kwArgs, requestOptions, headers, requestUrl) {
+			return lang.mixin({}, this.requestOptions || {}, requestOptions || {});
```

Comment:
I am pretty sure the `|| {}`s here are not necessary (`mixin` should default to just skipping over null/undefined things)